src/test/playwright/e2e/atlas/LearningPathManagement.spec.ts

```diff
@@ -0,0 +1,98 @@
+        await lpCheckbox.click();
+        await page.locator('#save-entity').click();
+
+        await page.goto(`/course-management/${course.id}/learning-path-management`);
+
+        // Assert: management UI becomes visible after enabling
+        await expect(page.locator('.learning-paths-analytics-container')).toBeVisible();
+    });
+
+    test('Instructor disables learning paths via course settings', async ({ page }) => {
+        await page.goto(`/course-management/${course.id}`);
+
+        await page.getByRole('link', { name: 'Settings' }).click();
+
+        // Toggle checkbox off and save
+        const lpCheckbox = page.locator('#field_learningPathsEnabled');
+        await expect(lpCheckbox).toBeVisible();
+        if (await lpCheckbox.isChecked()) {
+            await lpCheckbox.click();
+        }
+        await page.locator('#save-entity').click();
+
+        await page.goto(`/course-management/${course.id}/learning-path-management`);
+        await expect(page.locator('jhi-feature-activation')).toBeVisible();
+    });
+
+    test('Create simple learning path', async ({ page, courseManagementAPIRequests }) => {
+        // Enable learning paths first
+        await page.goto(`/course-management/${course.id}/learning-path-management`);
+        const activationCard = page.locator('jhi-feature-activation');
+        await expect(activationCard).toBeVisible();
+        await activationCard.locator('.jhi-btn').first().click();
+        await expect(page.locator('.learning-paths-analytics-container')).toBeVisible();
+
+        // Create all competencies and prerequisites via API and store their IDs
+        const comp1 = await courseManagementAPIRequests.createCompetency(course, 'Competency 1', 'Seed competency for learning path graph test 1');
+        const comp2 = await courseManagementAPIRequests.createCompetency(course, 'Competency 2', 'Seed competency for learning path graph test 2');
+        const comp3 = await courseManagementAPIRequests.createCompetency(course, 'Competency 3', 'Seed competency for learning path graph test 3');
+        const comp4 = await courseManagementAPIRequests.createCompetency(course, 'Competency 4', 'Seed competency for learning path graph test 4');
+        const comp5 = await courseManagementAPIRequests.createCompetency(course, 'Competency 5', 'Seed competency for learning path graph test 5');
+        const comp6 = await courseManagementAPIRequests.createCompetency(course, 'Competency 6', 'Seed competency for learning path graph test 6');
+        const prereq1 = await courseManagementAPIRequests.createPrerequisite(course, 'Prerequisite 1', 'Prerequisite of Competency 1');
+        const prereq2 = await courseManagementAPIRequests.createPrerequisite(course, 'Prerequisite 2', 'Prerequisite of Competency 2');
+
+        // Create all relations via API
+        // Note: In the UI "Requires" is shown, but the backend uses "ASSUMES" for the tail assuming the head
```

Comment:
```suggestion
        // Create all relations via API
        // Note: In the UI "Requires" is shown, but the server uses "ASSUMES" for the tail assuming the head
```
Use software keywords